config/crd/bases/catalogd.operatorframework.io_packages.yaml

```
@@ -18,7 +17,7 @@ spec:
   - name: v1beta1
     schema:
       openAPIV3Schema:
-        description: Package
+        description: Package is the Schema for the packages API
```

Can we use apostrophe for possessives in the spec. 
```suggestion
        description: Package is the Schema for the package's API
```

---

This is meant to be plural (e.g. the resource name).